### ISSUE TITLE
[Feature] Finalize video call feature.

### ISSUE THREAD

**Issue by Vishwajith-Shettigar** [Vishwajith-Shettigar/NextGen#24]
In NextGen, we have a video call feature, but it is not yet fully implemented, and there are some bugs in it.

**Comment by PhilipJohn005** [comment]
Hi, can u assign me this issue #24

**Comment by kushwxha** [comment]
hey, can you please assign it to me?

**Comment by Vishwajith-Shettigar** [comment]
@kushwxha done.

**Comment by Vishwajith-Shettigar** [comment]
@kushwxha Sorry, I have to assign tasks on a first-come, first-served basis. Please take a look at the other open issues.

**Comment by kushwxha** [comment]
Yup, you can assign it to me.

**Comment by kushwxha** [comment]
hey @Vishwajith-Shettigar,
i have to make change in this repo??
https://github.com/Vishwajith-Shettigar/video-call-server-node.js

**Comment by Vishwajith-Shettigar** [comment]
> hey @Vishwajith-Shettigar, i have to make change in this repo?? https://github.com/Vishwajith-Shettigar/video-call-server-node.js

No, clone that server and run locally, and connect local server with nextgen app. Video call feature works but some bugs are there, fix all the bugs.

**Comment by kushwxha** [comment]
Ok, I'll check it out.
